src/views/components/global/AgriTable/index.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div>
+    <md-table v-model="users" md-sort="name" md-sort-order="asc" md-card>
```

Comment:
There is a way to make it so when you hover over a table entry it will be highlighted. Maybe take a look at the docs and see if we could use any of these features they offer